Output/output.txt

```diff
@@ -0,0 +1 @@
+This file is intentionally left here in order to include this directory in git.
```

Comment:
Do we *want* this directory in git? I don't think we need it in version control unless we're storing contents, and I can't imagine that we'd want to store output in the repo.  I'm guessing you did it to avoid having to have directory-making code in output_engine.py but I think we could just prefix the default filename instead of using a directory and more or less achieve the same effect.